feed/templates/memberships/list.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+
+% block content %}
```

Comment:
Missing a `{`. Make sure you test your code on your computer (locally) before requesting a PR

```suggestion
{% block content %}
```